samples/OpenIdConnectSample/Startup.cs

```diff
@@ -73,30 +77,34 @@ public void ConfigureServices(IServiceCollection services)
             });
         }
 
-        public void Configure(IApplicationBuilder app)
+        public void Configure(IApplicationBuilder app, IAuthenticationHandlerProvider authSchemes)
         {
             app.UseDeveloperExceptionPage();
             app.UseAuthentication();
 
             app.Run(async context =>
             {
+                var response = context.Response;
+
                 if (context.Request.Path.Equals("/signedout"))
                 {
-                    await WriteHtmlAsync(context.Response, async res =>
+                    await WriteHtmlAsync(response, async res =>
                     {
                         await res.WriteAsync($"<h1>You have been signed out.</h1>");
-                        await res.WriteAsync("<a class=\"btn btn-link\" href=\"/\">Sign In</a>");
+                        await res.WriteAsync("<a class=\"btn btn-default\" href=\"/\">Sign In</a>");
```

Comment:
Are these buttons doing anything different?  Should the href be different for SignIn?